### PR TITLE
LTI-199: Upgrade rails version to 6.1

### PR DESCRIPTION
**Pull request by Mariam05**
Necessary to upgrade the broker to 6.1

### CODE REVIEW

File: bin/rails [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+APP_PATH = File.expand_path('../test/dummy/config/application', __dir__)
 
 # Set up gems listed in the Gemfile.
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
-require 'bundler/setup' if File.exist?(ENV['BUNDLE_GEMFILE'])
+require "bundler/setup" if File.exist?(ENV["BUNDLE_GEMFILE"])
 
-require 'rails/all'
-require 'rails/engine/commands'
+require "rails/all"
```
Comment (jfederico) [inline bin/rails]: Why did you change to `"`? The double one is only recommended when using string interpolation

File: rails_lti2_provider.gemspec [hunk 2]
```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.files = Dir['{app,config,db,lib}/**/*', 'MIT-LICENSE', 'Rakefile', 'README.rdoc']
 
   s.add_dependency('ims-lti', '~> 2.3.2.1')
-  s.add_dependency('rails', '~> 6.0.3')
+  s.add_dependency('rails', '~> 6.1')
```
Comment (jfederico) [inline rails_lti2_provider.gemspec]: Can we remove the dependency so the gem works with both?

**Review by jfederico — CHANGES_REQUESTED** [review]

File: bin/rails [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+# installed from the root of your application.
 
 ENGINE_ROOT = File.expand_path('..', __dir__)
 ENGINE_PATH = File.expand_path('../lib/rails_lti2_provider/engine', __dir__)
+APP_PATH = File.expand_path('../test/dummy/config/application', __dir__)
 
 # Set up gems listed in the Gemfile.
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
-require 'bundler/setup' if File.exist?(ENV['BUNDLE_GEMFILE'])
+require "bundler/setup" if File.exist?(ENV["BUNDLE_GEMFILE"])
```
Comment (jfederico) [inline bin/rails]: Why did you change to "? The double one is only recommended when using string interpolation

**Review by jfederico — CHANGES_REQUESTED** [review]

File: bin/rails (hunk 1 above)
Comment (Mariam05) [reply bin/rails]: that was done automatically by the `rails app:update` command but I can change it back

**Review by jfederico — APPROVED** [review]
